data_fixes/AC107/cid_uri_mapping.xquery

```diff
@@ -0,0 +1,9 @@
+xquery version "3.0";
+declare namespace ead = "urn:isbn:1-931666-22-9";
+
+declare variable $EAD as document-node()+ := collection("/Users/heberleinr/Documents/aspace_helpers/data_fixes/AC107/EADs?recurse=yes;select=*.xml")/doc(document-uri(.));

```

Comment:
Is this collection correct? It looks like a path on your local machine?

Reply:
yes! the collection is split into 16 files so I pulled them into a local directory first.